apps/meteor/client/views/admin/apps/AppDetailsPage/tabs/AppStatus/TooltipOnHover.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box, PositionAnimated, AnimatedVisibility } from '@rocket.chat/fuselage';
+import React, { RefObject, useRef, useState, ReactElement } from 'react';
+
+type TooltipOnHoverProps = {
+	element: ReactElement;
+	tooltip: ReactElement;
+};
+
+export const TooltipOnHover = ({ element, tooltip }: TooltipOnHoverProps): ReactElement => {
+	const ref = useRef<Element>();
+	const [isHovered, setIsHovered] = useState(false);
+
+	return (
+		<>
+			<Box is={'span'} ref={ref} onMouseEnter={(): void => setIsHovered(true)} onMouseLeave={(): void => setIsHovered(false)}>
+				{element}
+			</Box>
+
+			<PositionAnimated
+				anchor={ref as RefObject<Element>}
```

Comment:
no need for this assertion if you type the ref correctly above :)

Reply:
Much better! Many thanks, @juliajforesti 👊 I've fixed it ✅